crates/mongodb-connector/src/main.rs

```diff
@@ -9,6 +9,9 @@ use std::error::Error;
 
 use mongo_connector::MongoConnector;
 
+#[macro_use]
+extern crate lazy_static;
```

Comment:
You don't need `macro_use` anymore. You can import macros in the modules where they are used, in this case with `use lazy_static::lazy_static`

Reply:
Ah nice one. Fixed.